dev/AppNotifications/AppNotificationManager.cpp

```diff
@@ -167,6 +168,27 @@ namespace winrt::Microsoft::Windows::AppNotifications::implementation
         }
     }
 
+    void AppNotificationManager::Register()
+    {
+        RegisterHelper(winrt::hstring{} /* displayName */, std::wstring{} /* iconFilePath */);
```

Comment:
This is OK with empty values?

Reply:
What this basically means is that when both parameters are empty we fallback to gleaning the icon off process or shortcuts. If everything fails we fallback to default.

Thanks.